### DISCUSSION TITLE
Billing for a Team account for 25 members

### DISCUSSION THREAD

**Discussion by vijayideya** [community/community#125992] [Discover: GitHub Best Practices]
I chose GitHub for my team of 25 members. Last month's bill amounted to $116. Where can I locate the breakdown of my invoice? Is $16 designated for taxes?

**Comment by arjun-thecoder** [comment]
To locate the breakdown of your GitHub invoice, including any details about taxes, you can follow these steps:

### Log in to GitHub: Sign in to your GitHub account.

### Go to Billing Settings:
- Click on your profile picture in the top right corner of GitHub.
- Select "Settings" from the dropdown menu.
- In the settings sidebar, click on "Billing & plans" under the "Account settings" section.

### View Billing Details:
- Under "Billing & plans", you should see a section for "Invoices" or "Billing history".
- Click on "Invoices" to view a list of your past invoices.

### Download or View Invoice:
- Find the invoice for the last month (the one amounting to $116).
- Click on the invoice number or download the invoice as a PDF.

The detailed invoice should provide a breakdown of the charges, including any taxes applied. Look for a specific line item that mentions "tax" or "VAT" to determine if $16 was designated for taxes.

**Reply by vijayideya** [reply]
In invoice there were no detailed information about tax

**Comment by goldieowner17** [comment]
Thank you kindly for reaching out. If you need to report GitHub account-related abuse, you can reach out to our private support team. We are unable to assist with or discuss personal account details here in Discussions.

We hope that helps!
